_posts/2020-10-27-solidity-0.8.x-preview.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.8.x Preview Release'
+date: '2020-10-26'
+author: Solidity Team
+category: Releases
+---
+
+The Solidity team is currently preparing the changes for the next breaking release 0.8.x.
+
+We want to provide a preview release binary for everyone to try out so that you can give your feedback.
+
+The main change for 0.8.x is the switch to checked arithmetic operations by default.
+This means that ``x + y`` will throw an exception on overflow. In other words: You do not need SafeMath anymore!
+
+The scope of 0.8.x is not fully specified, so if you want to influence what goes in and how,
+then either [create a new issue](https://github.com/ethereum/solidity/issues/new?template=feature_request.md)
+or voice your opinion on an [existing issue](https://github.com/ethereum/solidity/issues?q=is%3Aissue+is%3Aopen+label%3A%22breaking+change+%3Awarning%3A%22)
+
+The current tentatively accepted changes are those in the "Implementation Backlog" column of our
+[project board](https://github.com/ethereum/solidity/projects/43?card_filter_query=label%3A%22breaking+change+%3Awarning%3A%22#column-9174474)
+that are labeled "Breaking Change".
+
+You can already browse the [updated documentation](https://solidity.readthedocs.io/en/breaking/),
+which should be consistent with the behaviour of the compiler. If you find any error, plaese
+open an issue!
+
+### Binaries!
+
+You can find the binaries at the following locations:
+
+ - [soljson.js](/data/08preview/soljson.js)
+ - [solc-macos](/data/08preview/solc-macos)
+ - [solc-static-linux](/data/08preview/solc-static-linux)
+ - [solc.exe](/data/08preview/solc.exe)
+
+Note that all of these compilers will warn about them being a prerelease version, which is intentional.
+
+If you use solc-js, just copy the binary as ``soljson.js`` into the root directory of the solc-js npm module.
+
+You can try out the compiler in [remix](https://remix.ethereum.org/#version=https://solidity.ethereum.org/data/08preview/soljson.js). If you use this link, the preview compiler is already selected.
+Take care to use `pragma solidity >0.7.0` - otherwise remix might switch back to a different version.
+
+For other ways to compile Solidity contracts, copy the binaries to the appropriate places.
+
+
+### Checked Arithmetic
+
+The "Checked Arithmetic" feature of Solidity 0.8.x consists of three sub-features:
+
```

Comment:
```suggestion
- Revert on assertion failures and similar conditions instead of using the invalid opcode.
- Actual checked arithmetic.
- ``unchecked`` blocks.

```